### ISSUE TITLE
Change license to one that is more applicable

### ISSUE THREAD

**Issue by hlapp** [phyloref/specification#5]
The MIT license is for software and not a good fit for different creative works. I'd suggest we either use CC-BY or CC-Zero. However, note that the W3C seems to be using their own [document license](https://www.w3.org/Consortium/Legal/2015/doc-license), which in spirit is more like CC-BY-ND.

**Comment by gaurav** [comment]
Good point! My instinct would be to go for CC-BY. Is there a good reason to prohibit derivative works? Here's what Creative Commons says about [adaptations](https://creativecommons.org/faq/#when-is-my-use-considered-an-adaptation) -- I don't think there's anything in there that we'd like to prohibit.

**Comment by hlapp** [comment]
I agree. I think it'd be best to check against TDWG expectations, conventions, or customs, in particular if we eventually will pursue ratification by TDWG.
